runtime/protocol/http-client/native/test/aws/smithy/kotlin/runtime/http/interceptors/RequestCompressionInterceptorTestNative.kt

```diff
@@ -5,5 +5,5 @@
 package aws.smithy.kotlin.runtime.http.interceptors
 
 internal actual fun decompressGzipBytes(bytes: ByteArray): ByteArray {
-    TODO("Not yet implemented")
+    TODO("Not yet implemented. Can we write a pure Kotlin implementation?")
 }
```

Comment:
**Comment**: We _can_ but we probably _shouldn't_ for performance and correctness reasons. We should look for existing platform implementations where possible. For Native that probably means binding to something like zlib or CRT.